charts/kyverno/values.yaml

```diff
@@ -216,11 +216,28 @@ config:
   - '[TokenReview,*,*]'
   - '[SubjectAccessReview,*,*]'
   - '[SelfSubjectAccessReview,*,*]'
-  - '[*,{{ include "kyverno.namespace" . }},kyverno*]'
```

Comment:
Per my other comment, either replace `kyverno*` with just `*` or maybe make that a new variable to accommodate a custom prefix that defaults to `kyverno` or use `{{ .Release.Name }}*`.

Reply:
Not that simple, there are a couple of templates involved here: fullname, serviceName, configmapName…

And even some that can be passed by the end user (serviceAccountName).

Going with a simple prefix is probably not going to be a viable option.